oio/xcute/jobs/container_mover.py

```diff
@@ -0,0 +1,152 @@
+# Copyright (C) 2019 OpenIO SAS, as part of OpenIO SDS
+#
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 3.0 of the License, or (at your option) any later version.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this library.
+
+from collections import Counter
+
+from oio.common.easy_value import boolean_value
+from oio.common.utils import cid_from_name
+from oio.directory.meta2 import Meta2Database
+from oio.rdir.client import RdirClient
+from oio.xcute.common.job import XcuteJob, XcuteTask
+
+
+class ContainerMoveTask(XcuteTask):
+
+    def __init__(self, conf, job_params, logger=None):
+        super(ContainerMoveTask, self).__init__(
+            conf, job_params, logger=logger)
+
+        self.src = job_params['service_id']
+        self.dst = job_params['dst']
+
+        self.meta2 = Meta2Database(conf, logger=logger)
+
+    def process(self, task_id, task_payload, reqid=None):
+        container_id = task_payload['container_id']
+
+        moved = self.meta2.move(container_id, self.src, dst=self.dst)
+
+        resp = Counter()
+
+        for res in moved:
+            if res['err'] is not None:
+                resp['errors'] += 1
+
+                continue
+
+            resp['moved_seq'] +=1
+            resp['to.' + res['dst']] +=1
+
+        return resp
+
+
+class ContainerMoveJob(XcuteJob):
+
+    JOB_TYPE = 'container-move'
```

Comment:
The user action isn't to move containers, but to decommission a meta2. We should use: `meta2-decommission`